src/components/modes/teacher/Response.js

```diff
@@ -145,13 +155,16 @@ class Response extends Component {
         <TableCell>{this.renderFeedbackCell()}</TableCell>
         <TableCell>
           <IconButton
-            data-cy="deleteButton"
+            data-id={buildDeleteButton(_id)}
+            data-cy={DELETE_RESPONSE_BUTTON_CYPRESS}
             color="primary"
             onClick={this.handleToggleConfirmDialog(true)}
           >
             <DeleteIcon />
           </IconButton>
           <ConfirmDialog
+            id={buildDeleteButton(_id)}
```

Comment:
Shouldn't this be another function? `buildConfirmButtonId` or something like that?